graphql/handler/transport/websocket.go

```diff
@@ -189,7 +191,15 @@ func (c *wsConnection) init() bool {
 			c.ctx = ctx
 		}
 
-		c.write(&message{t: connectionAckMessageType})
+		if initAckPayload != nil {
+			initJsonAckPayload, err := json.Marshal(*initAckPayload)
+			if err != nil {
+				panic(err)
```

Comment:
I'm not a huge fan of panicing, but seeing as it is used in other json marshalling  bits in the code here, this is fine.